pkg/kubectl/cmd/cmd.go

```diff
@@ -398,9 +414,7 @@ func HandlePluginCommand(pluginHandler PluginHandler, cmdArgs []string) error {
 	}
 
 	// invoke cmd binary relaying the current environment and args given
-	// remainingArgs will always have at least one element.
-	// execve will make remainingArgs[0] the "binary name".
```

Comment:
probably worthwhile to move this comment to the modified syscall.Exec invocation

Reply:
Done!